packages/desktop-client/src/hooks/useSpaceMeasure.tsx

```diff
@@ -0,0 +1,35 @@
+import { useRef } from 'react';
+
+import { View } from '../components/common/View';
+
+export const useSpaceMeasure = () => {
```

Comment:
Would existing `useViewportSize`/`useWindowSize` hooks achieve the same thing as this hook?